parsing/location.mli

```diff
@@ -73,6 +73,7 @@ val mkloc : 'a -> t -> 'a loc
 
 val input_name: string ref
 val input_lexbuf: Lexing.lexbuf option ref
+val input_phrase_buffer: Buffer.t option ref
```

Comment:
It would be nice to have a comment indicating that this is used by the error reporting code, and documenting that:

- if `!input_name` is `//toplevel//`, `!input_phrase_buffer` is `Some buf` where `buf` contains of the last toplevel phrase
- otherwise, `!input_phrase_buffer` is `None`.